internal/controllers/baremetalinstance/baremetalinstance_reconciler_function_test.go

```
@@ -0,0 +1,546 @@
+		catalogItemsClient := &fakeCatalogItemsClient{
+			getResponse: privatev1.BareMetalInstanceCatalogItemsGetResponse_builder{
+				Object: privatev1.BareMetalInstanceCatalogItem_builder{
+					Id:       catalogItemID,
+					Template: templateID,
+				}.Build(),
+			}.Build(),
+		}
+
+		t := &task{
+			r: &function{
+				logger:                          logger,
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)